src/services/DataService.ts

```diff
@@ -38,7 +38,7 @@ export class DataService {
     fetchWeatherStations = (): Promise<any> => {
         const STATIONS_FILTER_PREFIX = "items/weather_stations";
         const STATIONS_FILTER =
-            "?fields=id,name,location,website_url&fields=accuweather_location.current_weather_description,accuweather_location.isDayTime&filter[_and][0][_and][0][status][_eq]=published";
+            "?fields=id,name,location,website_url&fields=accuweather_location.current_weather_description,accuweather_location.isDayTime,accuweather_location.weather_condition_icon.asset&filter[_and][0][_and][0][status][_eq]=published";
```

Comment:
Related to the other 2 comments, not sure if it's missed or I got it wrong, but I think we can also remove `accuweather_location.isDayTime` from this call